ci/fih-tests_install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -x
+
+# Copyright (c) 2020 Arm Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# copy mcuboot so that it is part of the docker build context
+REPO_DIR=`pwd`
+mkdir -p ../../context
+pushd ../../context
+cp -r $REPO_DIR .
+cp -r $REPO_DIR/ci/fih_test_docker/execute_test.sh .
+cp -r $REPO_DIR/ci/fih_test_docker/build.sh .
+cp -r $REPO_DIR/ci/fih_test_docker/Dockerfile .
+./build.sh
+popd
+
+
```

Comment:
Please remove all those empty lines existing at the end of the files, including some of the other scripts and the Dockerfile.